test/sox_io_backend/test_info.py

```diff
@@ -32,9 +32,9 @@ def test_wav(self, dtype, sample_rate, num_channels):
         data = get_wav_data(dtype, num_channels, normalize=False, num_frames=duration * sample_rate)
         save_wav(path, data, sample_rate)
         info = sox_io_backend.info(path)
-        assert info.get_sample_rate() == sample_rate
-        assert info.get_num_frames() == sample_rate * duration
-        assert info.get_num_channels() == num_channels
+        assert info.sample_rate == sample_rate
+        assert info.num_frames == sample_rate * duration
+        assert info.num_channels == num_channels
```

Comment:
The functions are still in there so the change is not BC breaking, right? Is there a reason why you are updating the tests then?

Reply:
> The functions are still in there so the change is not BC breaking, right?

No. These `get_*` functions are removed so this is BC breaking.

Reply:
If a user tries to use them, what will happen? I don't see any instructions to the user on what to do?